src/applications/representatives/containers/Dashboard.jsx

```diff
@@ -2,6 +2,28 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { Link } from 'react-router';
 
+import { PoaRequestsWidget } from '../components/PoaRequestsWidget/PoaRequestsWidget';
```

Comment:
The PoaRequestsWidget is the default export with PoaRequestsWidget so its more standard to do:
`import PoaRequestsWidget from '../components/PoaRequestsWidget/PoaRequestsWidget';`